contracts/zero-ex/contracts/src/transformers/FillQuoteTransformer.sol

```diff
@@ -249,6 +257,17 @@ contract FillQuoteTransformer is
                     ).rrevert();
             }
         }
+
+        // Refund unspent protocol fees.
+        if (state.ethRemaining > 0 && data.refundReceiver != address(0)) {
```

Comment:
Would it would indicate an internal error when `state.ethRemaining > 0 && data.refundReceiver  == 0`? Wondering if we should `revert` in that scenario.

Reply:
No. That's the behavior we have now. The `PayTakerTransformer` will funnel the unspent ETH to the taker.